sr_robot_commander/doc/tutorial/ArmCommander.rst

```diff
@@ -1,60 +1,46 @@
 Arm Commander
 -------------
 
-Description
+Overview
 ~~~~~~~~~~~
 
-This commander provide commands specific to arm which allows to execute
-all actions of the `robot commander <RobotCommander.md>`__. Also it
-allows to move to certain position in Cartesian space, to joints states
-values and to move using certain trajectory.
+The SrArmCommander inherits all methods from the `robot commander <RobotCommander.html>`__ and provides commands specific to the arm. It allows to move to certain position in cartesian space, to a configuration in joint space
```

Comment:
I would say something like 'It allows movement to a certain position in cartesian space...'